src/main/java/io/github/thebusybiscuit/slimefun4/utils/WorldUtils.java

```diff
@@ -0,0 +1,29 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import javax.annotation.Nonnull;
+
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bukkit.World;
+
+/**
+ * Utilities for {@link World}.
+ *
+ * @author WalshyDev
+ */
+public final class WorldUtils {
+
+    private WorldUtils() {}
+
+    /**
```

Comment:
```suggestion
    /*
```

Reply:
Those are javadocs. 2 asteriks is correct...
The other instances where multiline-comments.